doc/minimal-manual.nix

```diff
@@ -11,5 +11,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ mmdoc ];
 
-  builder = writeScript "${name}-builder" "${mmdoc}/bin/mmdoc nixpkgs $src $out";
+  phases = [ "buildPhase" ];
+
+  buildPhase = ''
+    cp -r $src source
+    chmod -R u+w source
+    cp ${import ./doc-support/lib-functions-docs-cm.nix {}}/*.md source/functions/library/
+    ${mmdoc}/bin/mmdoc nixpkgs source $out
```

Comment:
is `buildInputs = [ mmdoc ];` even necessary when we refer to `mmdoc` by absolute path?

Reply:
Thanks; fixed!